src/index.html

```diff
@@ -2,12 +2,77 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="Apple iMac"
+        class="product__img"
+      >
+
+      <div class="product__title">
+        <h2>APPLE A1419 iMac 27" Retina<br>5K Monoblock (MNED2UA/A)</h2>
+      </div>
+
+      <div class="product__code">
+        Product code: 195434
+      </div>
+
+      <div class="product__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
```

Comment:
it would be better to do with `ul, li` tags
```suggestion
        <ul class="stars stars--4">
          <li class="stars__star"></li>
          ...
        </ul>
```